youtube_dl/options.py

```diff
@@ -179,6 +179,12 @@ def _scrub_eq(o):
         'Do not read the user configuration in ~/.config/youtube-dl/config '
         '(%APPDATA%/youtube-dl/config.txt on Windows)')
     general.add_option(
+        '--configfile',
+        dest='configfile', metavar='CONFIGFILE', type=str,
+        help='Do not read default configuration files found in '
+        '/etc/youtube-dl.conf and ~/.config/youtube-dl/config. '
```

Comment:
Is's absolutely pointless to clarify paths here. youtube-dl may be running on Window host where these paths make no sense at all.